examples/nlp/gpt/train_gpt_sft.py

```diff
@@ -119,15 +126,34 @@ def main(cfg) -> None:
     with open_dict(cfg):
         cfg.model.precision = cfg.trainer.precision
 
-    ptl_model = load_from_nemo(
+    assert cfg.model.peft.peft_scheme in ["lora", "none"], "Only support LoRA or Full finetuning"
+    if cfg.model.peft.peft_scheme == "lora":
+        assert (
+            cfg.model.optim.name != "distributed_fused_adam"
+        ), "LoRA doesn't support distributed_fused_adam, please use fused_adam"
```

Comment:
I think we shouldn't have to raise this on aligner in case nemo ever fixes it. @cuichenx will do a check for this on the nemo side. we can pick him his commit once done

Reply:
Cool, didn't notice the check in nemo

Reply:
605bad8baf13e6f5107f9e330f79b269fb8866dd